### DISCUSSION TITLE
Can this repository detects charts in the Gehler's Raw Dataset?

### DISCUSSION THREAD

**Discussion by tuvovan** [colour-science/colour-checker-detection#50] [General]
### Question

Hi, thank you for your work.

I wonder if this colour-checker-detection can run on RAW images? I'm trying to run it on this [dataset](https://www2.cs.sfu.ca/~colour/data/shi_gehler/)

Thanks!

**Comment by KelSolaar** [comment]
Hi @tuvovan,

The current detection code will not work on this dataset because the ColorChecker charts are small and not normal to the optical axis, thus even though it is trivial to read the images, you would not be able to extract the chart values.

![image](https://user-images.githubusercontent.com/99779/164799712-64b04d24-df60-43b2-9ce5-ffecbcd42b2f.png)

Cheers,

Thomas

**Comment by tuvovan** [comment]
@KelSolaar  Hi, Is there anyway around? We don't need to detect all of the patches in the plate anyway.

**Reply by KelSolaar** [reply]
You will be able to detect some swatches but no confidence as to which ones hence the difficulty to do anything after.
